assembl/templates/views/navBarRight.tmpl

```diff
@@ -20,6 +21,7 @@
     </li>
     {% endif %}
     {% endif %}
+    {% endif %}
```

Comment:
In this line:
```{% if request.current_route_path().split('/')[-1] not in ('login', 'register')  %}```
Maybe we should aim to put these kind of logic into methods? Hard to debug, read, etc. Eh, nitpicky..